src/data/quizdata.ts

```diff
@@ -0,0 +1,297 @@
+      { optionText: "882 million km", isCorrect: false },
+      { optionText: "228 million km", isCorrect: true },
+    ],
+  },
+  {
+    questionText: "How long does it take light to travel from the Sun to Mars?",
+    options: [
+      { optionText: "11 minutes", isCorrect: false },
+      { optionText: "13 minutes", isCorrect: true },
+      { optionText: "31 minutes", isCorrect: false },
+      { optionText: "1 minute", isCorrect: false },
+    ],
+  },
+  {
+    questionText:
+      "One astronomical unit (AU), is the distance from the Sun to Earth. How many AU units is Mars from the Sun?",
+    options: [
+      { optionText: "0.5", isCorrect: false },
+      { optionText: "1", isCorrect: false },
+      { optionText: "1.5", isCorrect: true },
+      { optionText: "2", isCorrect: false },
+    ],
+  },
+  {
+    questionText:
+      "What is Mars' axis of rotation / tilt with respect to the plane of its orbit around the Sun?",
+    options: [
+      { optionText: "5 degrees", isCorrect: false },
+      { optionText: "2 degrees", isCorrect: false },
+      { optionText: "25 degrees", isCorrect: true },
+      { optionText: "52 degrees", isCorrect: false },
+    ],
+  },
+  {
+    questionText:
+      "Mars has an elliptical (egg-shaped) orbit around the Sun. Which season in the northern hemisphere is the longest season at 194 sols?",
+    options: [
+      { optionText: "Winter", isCorrect: false },
+      { optionText: "Spring", isCorrect: true },
+      { optionText: "Summer", isCorrect: false },
+      { optionText: "Autumn", isCorrect: false },
+    ],
+  },
+  {
+    questionText:
+      "Mars has an elliptical (egg-shaped) orbit around the Sun. Which season in the southern hemisphere is the longest season at 194 sols?",
+    options: [
+      { optionText: "Winter", isCorrect: false },
+      { optionText: "Spring", isCorrect: false },
+      { optionText: "Summer", isCorrect: false },
+      { optionText: "Autumn", isCorrect: true },
+    ],
+  },
+  {
+    questionText:
+      "Mars has an elliptical (egg-shaped) orbit around the Sun. Which season in the southern hemisphere is the shortest season at 142 sols?",
+    options: [
+      { optionText: "Winter", isCorrect: false },
+      { optionText: "Spring", isCorrect: true },
+      { optionText: "Summer", isCorrect: false },
+      { optionText: "Autumn", isCorrect: false },
+    ],
+  },
+  {
+    questionText:
+      "Mars has an elliptical (egg-shaped) orbit around the Sun. Which season in the northern hemisphere is the shortest season at 142 sols?",
+    options: [
+      { optionText: "Winter", isCorrect: false },
+      { optionText: "Spring", isCorrect: false },
+      { optionText: "Summer", isCorrect: false },
+      { optionText: "Autumn", isCorrect: true },
+    ],
+  },
+];
+
+export const getRandomQuestions = (): Question[] => {
+  return questions
+    .map((q) => ({ q, sort: Math.random() }))
+    .sort((a, b) => a.sort - b.sort)
+    .map(({ q }) => q);
+};
```

Comment:
I suspect you'll only need one of these functions - I would recommend getting rid of this one and calling the other `getRandomQuestions` instead